### PR TITLE
Adding support code for maple finance 

### PR DESCRIPTION
**Pull request by ayushkumar63123**
Please review and tell if there are any issues with this PR.
Thanks

### CODE REVIEW

File: defitrack-protocols/Maple/pom.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>aave</artifactId>
```
Comment (TheMoneyLisa) [inline defitrack-protocols/Maple/pom.xml:6]: needs to be called maple. Also, please make the folder also lowercase.

File: defitrack-protocols/Maple/src/main/java/io/defitrack/protocol/Maple/MapleV2MainnetService.kt [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import org.springframework.stereotype.Component
+
+@Component
+class MapleV2MainnetService(
+    graphGatewayProvider: TheGraphGatewayProvider,
+    private val objectMapper: ObjectMapper,
+) {
+
+    val thegraph =
+        graphGatewayProvider.createTheGraphGateway("https://api.thegraph.com/subgraphs/name/maple/protocol-v3")
```
Comment (TheMoneyLisa) [inline defitrack-protocols/Maple/src/main/java/io/defitrack/protocol/Maple/MapleV2MainnetService.kt:17]: can't find this graph

Comment (TheMoneyLisa) [reply defitrack-protocols/Maple/src/main/java/io/defitrack/protocol/Maple/MapleV2MainnetService.kt:17]: I'm also pretty sure maple doesn't have the same structure as Aave, so not sure what the contracts you copied from the Aave protocol are referring to.

**Review by TheMoneyLisa — CHANGES_REQUESTED** [review]

**Comment by ayushkumar63123** [comment]
Sorry I am fixing all this and making the required changes ASAP thanks for
the quick review

**Comment by ayushkumar63123** [comment]
I have made the necessary changes please tell if they are wrong or something more is required

**Comment by ayushkumar63123** [comment]
@TheMoneyLisa please review updated pr

File: defitrack-protocols/Maple/pom.xml (hunk 1 above)
Comment (ayushkumar63123) [reply defitrack-protocols/Maple/pom.xml]: done this

File: defitrack-protocols/Maple/src/main/java/io/defitrack/protocol/Maple/MapleV2MainnetService.kt (hunk 2 above)
Comment (ayushkumar63123) [reply defitrack-protocols/Maple/src/main/java/io/defitrack/protocol/Maple/MapleV2MainnetService.kt]: I changed everything and created fresh folders with fresh files after a bit of research into what's needed here

**Comment by ayushkumar63123** [comment]
@TheMoneyLisa Its been a long time please review my new PR

**Review by TheMoneyLisa — CHANGES_REQUESTED** [review]
Hi, thanks for the quick feedback. Unfortunately, this code doesn't build, is incomplete and actually does not fetch any maple lending markets.
[Please visit our contribution guide to get started](https://docs.defitrack.io/developers/contributing).

**Comment by TheMoneyLisa** [comment]
Are you planning on continuing your work here?

**Comment by ayushkumar63123** [comment]
sorry i confused this as easy task but its not and the pay is also little compared to the difficulty level

**Pull request closed without merging**